kolibri/plugins/management/assets/src/vue/user-page/index.vue

```diff
@@ -1,24 +1,81 @@
 <template>
 
-  <div class="roster">
-    <div>
-      <user-create-modal></user-create-modal>
-    </div>
-    <div class="user-roster">
-      <ul>
-        <li v-for="user in getUsers">
-          <div>{{ user.last_name + ", " + user.first_name + ", "}} {{user.roles.length ? user.roles[0].kind : "user" }}</div>
-          <user-edit-modal :userid="user.id" :roles="user.roles" :username="user.username" :firstname="user.first_name" :lastname="user.last_name"></user-edit-modal>
-          <button @click="deleteUser(user.id)">Delete</button>
-        </li>
-      </ul>
+  <div class="user-roster">
+
+    <h2>
+      {{userPageTitle}}
+    </h2>
+
+    <span> ( {{ visibleUsers.length }} )</span>
+
+    <div class="toolbar">
+      <select v-model="roleFilter" name="user-filter">
```

Comment:
TODO: Accessible `select` element. See the [Slack discussion](https://learningequality.slack.com/archives/dev/p1465516856000314) for details.